sda-commons-server-jackson/src/test/java/org/sdase/commons/server/jackson/test/NestedNestedResource.java

```diff
@@ -0,0 +1,14 @@
+package org.sdase.commons.server.jackson.test;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.hibernate.validator.constraints.NotEmpty;
```

Comment:
Is it intended to use the deprecated Hibernate version? Should we move to `javax.validation.constraints.NotEmpty`?

Reply:
Correct. I added a commit that changes all the deprecated usages from `org.hibernate.validator.constraints.NotEmpty` to `javax.validation.constraints.NotEmpty`. That reduces the risk to inherit deprecated code via copy-and-pasting in the future.